src/Slider.jsx

```diff
@@ -568,6 +570,8 @@ class Rheostat extends React.Component {
         ? handlePos[idx - 1] + handleDimensions
         : SliderConstants.PERCENT_EMPTY, // 0% is the lowest value
     );
+
+    return validateMove ? validateMove(idx, nextPos) : nextPos;
```

Comment:
I don't think we should proxy through the return value from `validateMove`; we should be sanitizing and validating it somehow.

Reply:
We can parseFloat the return to ensure that we're getting back a valid percent and then run it through the main body of validatePosition. Does that work?

If we get NaN back from parseFloat(validateMove), should we throw, discard it and use the original, or do something else?

Reply:
I'd expect an exception.

Reply:
However, I'd also expect a function named "validate" to return either true/false, or `null`/an array of errors/an error - not a float.

Reply:
Both of the existing "validate" functions in the slider return corrected values, that's what I was basing my naming off of. Open to naming suggestions, and open to renaming the existing methods for clarity as well.

Reply:
We could rename all `validate` to `adjust` -> `adjustValues`, `adjustPosition` and `adjustMove`?